main.js

```diff
@@ -0,0 +1,187 @@
+// 1. Реалізувати фунцію-конструктор Автомобіля. Можливості автомобіля - почати поїздку, закінчити поїздку
+// 3. Реалізувати 3 фунції-конструктори - Хетчбек, Універсал, Седан. Їхнім прототипом - має бути Автомобіль та його можливості відповідно
+// 4. Окрім можливостей Автомобіля, ці 3 сутності повинні мати свої власні дані - розхід топлива, тип двигуна, об'єм двигуна, модель, рік випуску, маса, статус (або поїздка або готовий до поїздки) та справність авто
+// 5. Статус авто - не можна почати нову поїздку не завершивши попередню. Статус має бути Boolean значенням. Де true - це Автомобіль зараз у поїздці. Та false - Автомобіль стоїть, можна починати нову поїздку
+// 6. Відповідно методи Автомобіля почати та закінчити поїздку змінюють ці статуси у конкретного екземпляра
+// 7. Справність авто - це число від 0-100. Від кожної поїздки на авто (викликаний методом почати поїздку) - справність зменшується
+// 8. Кількість такої справності, яка зменшується визначати через Math.random. Випадкове число має бути в межах від 0.1 до 2.5
+// 9. Створити кілька автомоблів, поїздити. Переконатись, що кожна поїздка справді впливає на справність авто
+// 10. Створити функцію Дилер, яка приймає 1 аргумент - створений пунктом вище автомобіль (з його даними)
+// 11. Функція Дилер визначає скільки коштуватиме ремонт машини відштовхуючись від прейскуранту цін наведених у таблиці
+// 12. Дилер після ремонту має оновити справність конкретного авто до 100 та повідомити власнику ціну ремонту
+
+function Auto(){
```

Comment:
It works wrong
<img width="776" alt="image" src="https://user-images.githubusercontent.com/84010501/196045092-d81415a6-166f-4316-991f-b32ef26baef9.png">

Reply:
What exactly doesn't work? Ros said not to validate the data and assume that the user entered the data correctly. Did the function work correctly when you entered valid data?

Reply:
This is not a validation error, its just an execution of your code 
`const car_1 = new Hatchback(3.2, 'petrol', 2.6, 'Honda Civic', 2019, 1500, false);`